website_snippet_carousel_product/controllers/website.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020 Tecnativa - Alexandre Díaz
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import time
+
+from odoo import http
+from odoo.http import request
+
+from odoo.addons.website.controllers.main import QueryURL
+
+
+class Website(http.Controller):
+    @http.route(
+        ["/website/render_carousel_product"],
+        type="json",
+        auth="public",
+        website=True,
+        csrf=False,
+        cache=30,
+    )
+    def render_carousel_product(
+        self, template, domain=False, limit=12, pps=4, **kwargs
```

Comment:
What is `pps`? Would you mind to use a more descriptive name?

Reply:
1. Ok
2. Yes, but Odoo doesn't offer any domain selector for the frontend (or i can't see it)... For me the domain selector is outside the scope of this module.

PPS = Products Per Slide (Similar to the PPG (products per page) from Odoo)